public/swagger.v1.json

```diff
@@ -969,15 +969,12 @@
           "x-go-name": "OpenIssues"
         },
         "owner": {
-          "x-go-name": "Owner",
           "$ref": "#/definitions/User"
```

Comment:
Ack. That keeps getting put in. Why is it in the repo if automatically generated? Each time I have to remember to revert it. Forgot this time. Such a pain.